rockcraft.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE-rock file for licensing details.
+---
+name: charmed-valkey  # the name of your ROCK
+base: ubuntu@22.04  # the base environment for this ROCK
+version: '7.2.5'  # just for humans. Semantic versioning is recommended
+summary: Charmed Valkey ROCK OCI  # 79 char long summary
+description: |
+  This is an OCI image that bundles Valkey together with the metrics exporter
+  in order to be used in Charmed Operators, providing automated operations 
+  management from day 0 to day 2 on Valkey, on top of a Virtual Machine cluster 
+  and K8s cluster. It is an open source, end-to-end, production ready data 
+  platform on top of cloud native technologies.
+license: Apache-2.0  # your application's SPDX license
+
+platforms:  # The platforms this ROCK should be built on and run on
+  amd64:
+
+services:
+  valkey:
+    override: replace
+    startup: enabled
+    summary: Start Valkey
+    command: "valkey-server"
```

Comment:
I believe we should ensure that this ran with the `vakey` user not root

Reply:
done